lblod/ember-rdfa-editor-lblod-plugins
# GN-4818: Add missing translations for date picker

### Overview

`DateTimePicker` is missing translations, see image.

<img width="1275" alt="image" src="https://github.com/lblod/ember-rdfa-editor-lblod-plugins/assets/769698/c4473d14-2d7c-46ba-86fb-093d8b4e19d0">

##### connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4818

### Setup

1. Checkout
2. `npm run start`

### How to test/reproduce

Insert date variable, attempt to change the value for it, check that there are no errors in the console.

### Checks PR readiness
